ultra/tests/test_env.py

```diff
@@ -61,6 +63,22 @@ def run_experiment():
         self.assertTrue(task_id1 == task_id)
         self.assertTrue(task_level1 == task_level)
 
+    def test_scenario_sequence(self):
+        @ray.remote(max_calls=1, num_gpus=0, num_cpus=1)
+        def run_experiment():
+            _, env = prepare_test_env_agent(
+                scenario_info=[prebuilt_scenario1, prebuilt_scenario2]
+            )
+            scenarios = env.scenario_info
```

Comment:
Is it necessary to include a getter() for the scenario_info?

Reply:
We currently don't have a getter for the `scenario_info` in the ULTRA env. And I think it is fine if we continue to not have a getter as we only access the env's scenario info in this test as far as I'm aware. We can also obtain the `scenario_info` from the env's `info` attribute: https://github.com/huawei-noah/SMARTS/blob/2c1979e5b290c4267d8e3347e12ad4e2ff0727ef/ultra/ultra/env/ultra_env.py#L158-L164

If the env's `scenario_info` attribute is used elsewhere (other than the tests) then it might be good to make a getter (using the `@property` decorator).

Let me know your thoughts.

Reply:
Hmm, I do not know if we do use the `scenario_info` object elsewhere. I am a bit of keen to implement getters/setters when it comes to retrieving/modify class members, but I believe it is fine for now. Yes, we can also use the `info` attribute to get the scenario info if we need it.